K8s-deployment/Charts/databroker/example-secrets/secrets/init-config.json

```diff
@@ -160,9 +160,9 @@
         {
           "vhost": "IUDX",
           "permission": {
-            "configure": "^$",
-            "write": "^$",
-            "read": "^$"
+            "configure": ".*",
```

Comment:
Why is this modified to give all access when previous setting was to deny all access?
Please revert.